league/manager/models.py

```diff
@@ -25,64 +23,83 @@ def played_matches(self):
         return self.match_set.filter(result__isnull=False).count()
 
     @property
-    def ended(self):
+    def is_ended(self):
         if self.all_matches > 0:
             return self.all_matches == self.played_matches
         else:
             return False
 
     @property
     def league_winner(self):
-        #TODO: zrobic winnera ligi ASAP
+        # TODO: Implement condition for winning a league ASAP
         pass
 
 
 class Team(models.Model):
+    league = models.ForeignKey(League, on_delete=models.CASCADE)
```

Comment:
I like that change - means that you think about what is important in model. Great!